sdcm/loader.py

```diff
@@ -16,16 +16,21 @@
 
 from sdcm.utils import FileFollowerThread
 
+METRICS = {}
+
 
 class CassandraStressExporter(FileFollowerThread):
     def __init__(self, instance_name, metrics, cs_operation, cs_log_filename, loader_idx, cpu_idx):
         super(CassandraStressExporter, self).__init__()
         self.metrics = metrics
         self.cs_operation = cs_operation
         self.cs_log_filename = cs_log_filename
-        self.cs_metric = self.metrics.create_gauge('collectd_cassandra_stress_%s_gauge' % self.cs_operation,
-                                                   'Gauge for cassandra stress metrics',
-                                                   ['cassandra_stress_%s' % self.cs_operation, 'instance', 'loader_idx', 'cpu_idx', 'type'])
+        gauge_name = 'collectd_cassandra_stress_%s_gauge' % self.cs_operation
+        if gauge_name not in METRICS:
+            METRICS[gauge_name] = self.metrics.create_gauge('collectd_cassandra_stress_%s_gauge' % self.cs_operation,
```

Comment:
if we create metric with the same name, what happens?

Reply:
the second time we'll fail, and no metrics for the stress. I failed to test it with two loaders tp catch this issue.